### PR TITLE
#167826411 Add book a seat number functionality.

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?
Adds a book seat number functionality

#### Description of Task to be completed?
- Add HTML script to the book seat page.
- Add styling to the book seat HTML file. 

#### How should this be manually tested?
Clone this repository and run `npm install && run test`

#### Any background context you want to provide?
Adding a book seat number functionality allows users to book which seat they prefer among the not booked seats.

#### What are the relevant pivotal tracker stories?
#167826411

#### Screenshots (if appropriate)
Screenshot from 2019-08-12 12-15-24

### CODE REVIEW

File: UI/html/specifictrip.html [hunk 1]
```diff
@@ -46,7 +46,6 @@
             <td>KDE 463E</td>
         </tr> 
     </table>
-
-    <button class="book-button">Book A Seat</button>
+    <a href="bookseat.html" class="book-button">Book A Seat</a>
 </body>
 </html>
```
Comment (oesukam) [inline UI/html/specifictrip.html]: @nziokaivy Blank line required

File: UI/js/index.js [hunk 2]
```diff
@@ -43,4 +43,42 @@ function openTab(evt, tabName) {
...
+      if (td) {
+        txtValue = td.textContent || td.innerText;
+        if (txtValue.toUpperCase().indexOf(filter) > -1) {
+          tr[i].style.display = "";
+        } else {
+          tr[i].style.display = "none";
+        }
+      }       
+    }
   }
```
Comment (oesukam) [inline UI/js/index.js]: @nziokaivy Blank line required

**Review by oesukam — CHANGES_REQUESTED** [review]
Great job. You just have to add blank lines at the of you files and remove unnecessary spaces from your HTML files.

**Comment by nziokaivy** [comment]
> Great job. You just have to add blank lines at the of you files and remove unnecessary spaces from your HTML files.

@oesukam I have now removed all the unwanted spaces and add blank lines at the end of files

**Comment by oesukam** [comment]
@nziokaivy You have merge conflict under your `package.json` file that you forgot to resolve

**Review by oesukam — APPROVED** [review]

**Pull request merged**